jetty-client/src/main/java/org/eclipse/jetty/client/http/HttpConnectionOverHTTP.java

```diff
@@ -142,10 +142,14 @@ public boolean onIdleExpired()
         long idleTimeout = getEndPoint().getIdleTimeout();
         boolean close = delegate.onIdleTimeout(idleTimeout);
         if (close)
-            close(new TimeoutException("Idle timeout " + idleTimeout + " ms"));
+            close(new TimeoutException("Idle timeout " + idleTimeout + " ms: " + describe(getEndPoint())));
         return false;
     }
 
+    private String describe(EndPoint ep) {
+        return ep.getLocalAddress() + "<->" + ep.getRemoteAddress();
```

Comment:
Why not just use EndPoint.toString.  I know it gives a lot more information and is a bit verbose, but that information may be useful in diagnosis.   Also will handler the case where the endpoint type does not have a remote or local address